static/src/javascripts/projects/common/modules/identity/okta.ts

```diff
@@ -0,0 +1,65 @@
+import type { CustomClaims, IdentityAuthState } from '@guardian/identity-auth';
+import { IdentityAuth } from '@guardian/identity-auth';
+
+// the `id_token.profile.theguardian` scope is used to get custom claims
+export type CustomIdTokenClaims = CustomClaims & {
+	email: string;
+	braze_uuid: string;
+};
+
+const getStage = () => {
+	return window.guardian.config.page.stage;
```

Comment:
Could this just be set as a `const`?

Reply:
(as a note - this is non-blocking as it's all self contained and makes sense, just an interesting chat!)